libs/pages/elewa/careers/src/lib/components/careers-hero/careers-hero.component.html

```diff
@@ -0,0 +1,21 @@
+<div id="careers-hero"> 
+    <div class="container">
+        <div class="content">
+            <h1 class="title">
+                Working at Elewa
+            </h1>
+            <br>
+            <p class="paragraph">
+                Elewa Group values education and innovation. As a tech company, <br>
+                we help employees grow and reach their full potential. Join a <br>
+                talented and driven team and make a real impact . Take your <br>
+                career to new heights with Elewa.
```

Comment:
Just restrict the paragraph width to about 60 - 70% instead of using the <br> tag. This will force the ext to move to next line

Reply:
Thanks, I have added the paragraph width.